### DISCUSSION TITLE
How to start dubbo whithout ApplicationEvent  in spring-boot

### DISCUSSION THREAD

**Discussion by SpinSi** [apache/dubbo#14081] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar issues.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Dubbo Version

Dubbo version 3.1.11

### Steps to reproduce this issue

When i study dubbo deploy code , I fount the way to run dubbo in  DubboDeployApplicationListener.java , It means it starts by ApplicationEvent . 
Is any way to run dubbo without  DubboDeployApplicationListener.java .  because we need to run dubbo order by some list 
like   middlewareA -> middlewareB -> Dubbo -> Rocket -> SomeJob  .....   . 

### What you expected to happen

I mean the controler of start /stop dubbo  should be application coder.  That is useful  to deploy applicaiton graceful !

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by finefuture** [comment]
You can use the DubboBootstrap API.

**Comment by SpinSi** [comment]
DubboBootstrap API is a way , but it is not a good way . Spring is already used in dubbo . Suggest adding ModulDeplayer.java default Sping bean , adding an automatic startup configuration (ex: dubbo.auto.start ), which defaults to true to start , coder can set false to control it

**Comment by SpinSi** [comment]
Dubbo  support a  configuration named delay to delay export dubbo rpc service ，but the time is unfixed, sometimes the delay is invalid.  I think initialization and startup of Dubbo are two separate steps that should be handed over to the user for control.  So an automatic startup configuration (ex: dubbo.auto.start  default true ) is needed.
